addons/im_livechat/static/src/core/web/suggestion_service_patch.js

```diff
@@ -1,13 +1,40 @@
 import { SuggestionService } from "@mail/core/common/suggestion_service";
+import { cleanTerm } from "@mail/utils/common/format";
 
 import { patch } from "@web/core/utils/patch";
 
 patch(SuggestionService.prototype, {
     /** @override */
-    getSupportedDelimiters(thread) {
+    getSupportedDelimiters(thread, composer) {
```

Comment:
I don't think we could as `thread` holds the main `composer`, but it might be possible that someone can edit the message and there he tries to use command. So, at that time it will use the record of the main composer instead of the composer linked with the message.

Reply:
I don't think it makes sense to edit a message to trigger a command, especially since the message post flow is interrupted if a command is detected. In that case, what would be the edit behavior? Cancel the update and trigger the command? That doesn't seem good either in my opinion. Additionally, this feature does not currently work because there is no code to handle such a scenario in the message edit flow. I believe we should consider disabling commands when editing a message. Perhaps we can seek @seb-odoo's opinion on this matter.

Reply:
I agree, previous messages are not meant to be edited in a command.